### ISSUE TITLE
User Stories

### ISSUE THREAD

**Issue by rbeeler** [mck368/AD320_FinalProject#1]
**User Stories**

-v indicates a viable user story
-o indicates an optional user story
Format: Topic (HTML Page)

##############################################################################
**'*' (All Pages)**
-v As a developer, I want to be able to make a single change to the header/footer and have it affect all pages.
-v	 As a developer, I want the styles of the pages to be similar. 

##############################################################################
**Home page (Index)**
-v	 As a user, I want to be able to click the movie logo to get to the home page. 
-o	 As a user, I want to be able to see my friends.
-o	 As a user, I want to see recent blog entries.
-o	 As a user, I want to see recommended movies from my friends. 
	AND
-o	As a user, I want to see recommended movies similar to movies in my list.

##############################################################################
**Movies Database (Movies)**
-v	As a user, I want to be able to read all items so that I am able to decide what I want to get from a list.
-v	As a user, I want to be able to search for an item in the database. 
	AND
-v	As a user, I want to have reasonable results for my search in the database. 
-v	As a user, I want to click on an item, and get more information about it. 
	AND
-v	As a user, I want to be able to see the image of the movie.
	AND 
-o	As a user, I want to be able to click the image of the movie to reveal more details, or go to an IMDB page.
-o	As a user, I want to be able to rate a movie and have it affect the audience score.

##############################################################################
**My Movies (Movies)**  
-v	As a user, I want to be able to create an item so that I am able to make a list. 
	AND
-v	As a user, I want to be able to delete items so that I am able to remove from a list. 
	AND
-v	As a user, I want to be able to edit items so that I am able to change from a list. 

##############################################################################
**Friends Database (Friends)**
-v	As a user, I want to navigate easily between my lists so that I can personalize my space easily.  
-o	As a user, I want to be able to "like" my friends' movie choices.
-o	As a user, I want to recommend movies to my friends.
-o	As a user, I want to be able to see all of my friends' "My Movies" pages.

##############################################################################
**Blog Database (Blog)** 
-v	As a user, I want to be able to link to social media sites from the web page.
-o	As a user, I want to be able to send messages to my friends.

##############################################################################
**Form Page (Contact Us)**
-v	As a user, I want to be able to send feedback to the website, or ask questions. 

##############################################################################

**Comment by tcoducayen** [comment]
There are 24 user stories here, 14 are viable, 10 are optional

Movies database: I think that this should not be able to be touched by users, except for audience score
My Movies: I think this should be a seperate page
Blog: I don't think this page is necessary, but can easily move into the "my movies" page
Contact Us: I think we should have this page, but make it in the footer and not in the navigator

I think this format looks easier to tell where all of the changes are going to be placed. 
I hope it is easy to read and understand! 

@rbeeler - I took most of your user stories and reorganized them and added more, feel free to change it back if this doesn't look okay.

**Comment by adslin** [comment]
According project requirement said that there is no hard rules on the number of stories for this, but it should be a reasonable estimate of the minimum amount of work you expect your group to accomplish. However, we can follow this list, and set up functions in each page.

**Comment by tcoducayen** [comment]
Yeah! Well I figure a couple of these are done and a lot are optional, but feel free to turn off any you think are too difficult

**Comment by rbeeler** [comment]
Yes, not gonna lie, this is too much since we have to complete them all this time around. I think it is better to have a smaller list, and then if we have time to do more that will just be an extra bonus. Like recommended movies from my friends should probably be taken off lol.

**Comment by tcoducayen** [comment]
Hmm, I am wondering if it is just not clear, I tried to make it so the ones we must do have a -v next to them and the ones that are optional have a -o next to them. Sorry if they wasn't clear.  In the assignment though it said to list optional and viable assignments. Does that make sense?

**Comment by adslin** [comment]
That was an example which breaks down a large user stories into smaller tasks. We might only focus on mvp.text first, then we can work on others.

**Comment by tcoducayen** [comment]
Using this data, this is what mvp.txt would look like:
### mvp.txt starts here ###
As a developer, I want to be able to make a single change to the header/footer and have it affect all pages.

As a developer, I want the styles of the pages to be similar.

As a user, I want to be able to click the movie logo to get to the home page
.
As a user, I want to be able to read all items so that I am able to decide what I want to get from a list.

As a user, I want to be able to search for an item in the database.
AND
As a user, I want to have reasonable results for my search in the database.

As a user, I want to click on an item, and get more information about it.
AND
As a user, I want to be able to see the image of the movie.

As a user, I want to be able to create an item so that I am able to make a list.
AND
As a user, I want to be able to delete items so that I am able to remove from a list.
AND
As a user, I want to be able to edit items so that I am able to change from a list.

As a user, I want to navigate easily between my lists so that I can personalize my space easily.

As a user, I want to be able to link to social media sites from the web page.

As a user, I want to be able to send feedback to the website, or ask questions.
### end of mvp.txt ###

Note: all I did was take the optional stories out.  There are 14 core, viable stories for out minimum viable product.  The optional user stories are simply used to point out if we want more functionality and what would be "nice to have" as suggested in the fifth bullet of the assignment.

If you think there are things listed here that are not viable, feel free to remove them from the list and change them to optional in the above text.
